br/pkg/restore/client.go

```diff
@@ -1237,15 +1238,18 @@ func (rc *Client) IsSkipCreateSQL() bool {
 // 2. create table/create and rename table: the first create table will lock down the id cache.
 // because we cannot create onExistReplace table.
 // so the final create DDL with the correct auto increment/random id won't be executed.
-func (rc *Client) DDLJobsMap() map[UniqueTableName]bool {
-	m := make(map[UniqueTableName]bool)
+func (rc *Client) GenDDLJobsMap() {
```

Comment:
```suggestion
func (rc *Client) GenerateDDLJobtsMap() {
```

the name is too close to `GetDDLJobsMap`

Reply:
thanks, reasonable to rename it.